### PR TITLE
03 unit testing

### PR DESCRIPTION
**Pull request by River2015**

### CODE REVIEW

File: src/app/core/components/courses/components/course/course.component.spec.ts [hunk 1]
```diff
@@ -1,25 +1,161 @@
...
+    spyOn(console, 'log');
+    component.edit();
+    expect(console.log).toHaveBeenCalled();
+  });
+
+  it('should raise id for deleting  event when clicked', () => {
+    let deletedId: 1;
+    component.courseDelete.subscribe((id: 1) => deletedId = id);
+    component.delete();
+  });
```
Comment (bye) [inline src/app/core/components/courses/components/course/course.component.spec.ts:121]: I don't see expect in this test

File: src/app/core/core.module.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ import {CourseComponent} from './components/courses/components/course/course.com
   ],
   imports: [
     CommonModule
-  ]
+  ],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```
Comment (bye) [inline src/app/core/core.module.ts]: why do you still need CUSTOM_ELEMENT_SCHEMA?

File: src/app/core/components/courses/components/course/course.component.spec.ts (hunk 1 above)
Comment (River2015) [reply src/app/core/components/courses/components/course/course.component.spec.ts:121]: added

File: src/app/core/core.module.ts (hunk 2 above)
Comment (River2015) [reply src/app/core/core.module.ts]: deleted

**Review by bye — APPROVED** [review]

**Pull request merged**